### ISSUE TITLE
Chaos Break won't show audio files

### ISSUE THREAD

**Issue by Foxero18** [m35/jpsxdec#37]
jPSXdec version: 1.04
Operating system: 
Java runtime environment : 

 PlayStation Game 
----------------------------------

Name: Chaos Break
Region code : SCPS-45480

 Description 
-------------

I was looking through the untested PS1 games from my collection and Chaos Break has only one issue;

The audio files won't show up, I can view the TIM-files only. 
The FMV cutscenes also show just fine.

Here's the IDX of the game.
[ChaosBreak.zip](https://github.com/m35/jpsxdec/files/6370702/ChaosBreak.zip)

**Comment by m35** [comment]
[According to the PlayStation DataCenter](https://psxdatacenter.com/games/J/C/SCPS-45480.html) that disc has a CD audio track. jPSXdec doesn't handle those but there are other programs can read CD audio tracks out of disc images (e.g. [IsoBuster](https://www.isobuster.com/)). Could you give that a try?

**Comment by Foxero18** [comment]
> [According to the PlayStation DataCenter](https://psxdatacenter.com/games/J/C/SCPS-45480.html) that disc has a CD audio track. jPSXdec doesn't handle those but there are other programs can read CD audio tracks out of disc images (e.g. [IsoBuster](https://www.isobuster.com/)). Could you give that a try?

I actually did.
I was able to convert it to a WAV file, but if I'd happen to want to translate/Dub that file, wouldn't re-inserting it be whole other process?

I'm not sure if Isobuster is capable of doing that.

**Comment by m35** [comment]
Could you generate the index (.idx) of the disc and attach it to this issue? I'd like to take a look at it.

**Comment by Foxero18** [comment]
> Could you generate the index (.idx) of the disc with jPSXdec and attach it to this i? I'd like to take a look at it.

I updated the de

**Comment by Foxero18** [comment]
I accidentally closed the issue.

But I added the idx of the game.

**Comment by Foxero18** [comment]
> Could you generate the index (.idx) of the disc with jPSXdec and attach it to this issue? I'd like to take a look at it.

Can you still see the link?

**Comment by m35** [comment]
Yes thanks for adding it. This game makes a pretty good case for adding CD audio extraction/replacing to jPSXdec.... some day ;)

**Comment by Foxero18** [comment]
Yes, I'm very fascinated by how the Jpsxdec works.

It's a pretty neat tool ^^

The Seq/Vag support will be great once that becomes available too.
